src/_includes/header2.html

```diff
@@ -0,0 +1,53 @@
+{%- include google-tag-manager-body.html -%}
+
+<header class="siteHeader2">
+  <div class="siteHeader2-logoAndMenuButton">
+    {% if page.layout == 'landing' %}
+    <!-- For the "Guide To App Devs" - /welcome_to_tilt.html -->
+    <a class="imageLink imageLink--logo-imagemark" href="{{site.landingurl}}/">
+      {% svg assets/svg/logo-imagemark.svg %}
+    </a>
+    {% elsif page.layout == 'home' %}
+    {% svg assets/svg/Tilt-logo.svg class="siteHeader2-logoLink-svg" %}
+    {% else %}
+    <a class="siteHeader2-logoLink" href="{{site.landingurl}}/">
+      {% svg assets/svg/Tilt-logo.svg class="siteHeader2-logoLink-svg" %}
+    </a>
+    {% endif %}
+    <button class="siteHeader2-menuButton" onclick="document.querySelector('.siteHeader2').classList.toggle('has-openMenu')">
+      {% svg assets/svg/menu.svg class="siteHeader2-menuSvg" %}
+      {% svg assets/svg/x.svg class="siteHeader2-closeSvg" %}
+    </button>
+  </div>
+
+  <nav class="siteHeader2-nav">
+    <ul class="siteHeader2-navList">
+      {% for link in site.data.header2 %}
+        {% assign href = link.href %}
+        {% if link.landingurl %}
+          {% assign href = (href | prepend: "/" | prepend: site.landingurl) %}
+        {% elsif link.docsurl %}
+          {% assign href = (href | prepend: "/" | prepend: site.docsurl) %}
+        {% elsif link.blogurl %}
+          {% assign href = (href | prepend: "/" | prepend: site.blogurl) %}
+        {% endif %}
+
+        {% if link.github %}
+          <li class="siteHeader2-navItem">
+            <a class="siteHeader2-navLink--github" href="{{href}}">
```

Comment:
usually, in BEM, you only have the modifier class on the element if you also have the base class. So this would be class="siteHeader2-navLink siteHeader2-navLink--github"
http://getbem.com/naming/

Reply:
Oh! Duly noted. I think I was used to other BEM-adjacent systems that were a bit more loose with the use of modifiers.